packages/react-dom/src/server/ReactPartialRendererHooks.js

```diff
@@ -281,7 +281,9 @@ export function useLayoutEffect(
       "be encoded into the server renderer's output format. This will lead " +
       'to a mismatch between the initial, non-hydrated UI and the intended ' +
       'UI. To avoid this, useLayoutEffect should only be used in ' +
```

Comment:
I think grammar gets a bit clunky. It's almost a run-on sentence.

Reply:
damn, thanks for that. love it.

Reply:
done